README.md

```diff
@@ -1,29 +1,35 @@
 # DBOS Transact
 
-DBOS Transact is a **transactional TypeScript framework** for developing database-backed applications with built-in once-and-only-once code execution.
+DBOS Transact is a **modern TypeScript framework** for backend applications.
 
-You want to build your next application with DBOS Transact because:
+You want to build your next application with DBOS Transact because you need:
 
-- **It's reliable by default**.  If your workflows are interrupted for any reason, they [will always resume from where they left off](https://docs.dbos.dev/tutorials/workflow-tutorial#reliability-guarantees).  Reliable message delivery is [built in](https://docs.dbos.dev/tutorials/workflow-communication-tutorial#reliability-guarantees-1). Idempotency is [built in](https://docs.dbos.dev/tutorials/idempotency-tutorial).
-- **It's simple**.  Write your business logic with serverless functions and either [self-host them](https://docs.dbos.dev/tutorials/self-hosting) or [deploy them to DBOS cloud](https://docs.dbos.dev/getting-started/quickstart#deploying-to-dbos-cloud) in minutes. Store your data in any PostgreSQL-compatible database&mdash;we'll manage the transactions for you.
-- **It makes debugging easy**.  With our [time travel debugger](https://docs.dbos.dev/cloud-tutorials/timetravel-debugging), you can "rewind time" and replay any DBOS Cloud trace locally on your computer, exactly as it originally happened.
+- **Blazing-fast, developer-friendly serverless**.  Develop your project locally and run it anywhere. When you're ready, [deploy it for free to DBOS Cloud](https://docs.dbos.dev/getting-started/quickstart#deploying-to-dbos-cloud) and we'll host it for you, [25x faster](https://www.dbos.dev/blog/dbos-vs-aws-step-functions-benchmark) and [15x cheaper](https://www.dbos.dev/blog/dbos-vs-lambda-cost) than AWS Lambda.
```

Comment:
Would prefer these to all be nouns, wonder if it is worth splitting these 3 things up a bit.

Reply:
I would advocate for not putting advertisement for the commercial product in the first bullet point motivating the open source framework